es/index.md

```diff
@@ -1,110 +1,90 @@
-1. Click on the link to find all resources in a single page including an optional Power Point
-1. Select and modify the content to your needs, and present away!
+## 🧐 Suena interesante, ¿qué necesito hacer?
 
-## 🧐 Sounds interesting, what do I need?
-You only need to find an available title from a topic that you like and use any (or all) resources to create a presentation. No need to know Git or GitHub unless you want to fork this repository and make changes (✅ which is OK if you want to!).
+Lo único que necesitas es: encontrar un título disponible de un tema que te guste y usar uno (o todos) los recursos disponibles para crear tu presentación. No es necesario conocer Git o GitHub, a menos que desees bifurcar (fork) este repositorio y realizar cambios (lo cual esta super bien ✅, si es lo que deseas)
 
+## 🤔 Sigo sin comprender
+Mira este video corto que explica como tú puedes utilizar estas presentaciones.
 
-## 🤔 I'm still confused!
-Watch this short video that explains how to use the ready-to-use presentations
+> 🎥 Haz clic en la imagen inferior para aprender como presentar un taller de esta librería
 [![Workshop Library overview ](/images/how-to.gif)](https://youtu.be/tylrSBnjHGo "Workshop Library overview - Click to Watch!")
-> 🎥 Click the image above to learn how to deliver a workshop from this library
 
-## 🧠 AI and ML
-From building ML models using Custom Vision to creating a web application with Flask, this category is full of useful resources related to AI and ML so that you can present. Click on the available presentations below 👇
-
-  <details>
-    <summary>Intermmediate</summary>
+## 🧠 Inteligencia Artificial y Machine Learning
 
+Desde construir modelos de Machine Learning utilizando Custom Vision hasta la creación de una aplicación web con Flask, esta categoría está llena de recursos útiles relacionados con Inteligencia Artificial y Machine Learning que puedes presentar. A continuación, haz clic en las presentaciones disponibles 👇: 
 
-   - [Build a Machine Learning Model using Custom Vision](./full/ml-model-custom-vision/README.md)
-   - [Build an AI web app by using Python and Flask](./full/flask-text-translator/README.md)
-   - [Create a Regression Model with Azure Machine Learning designer](./full/regression-aml-designer/README.md)
-   - [Integrating Custom Vision with Power Apps for Diabetic Retinopathy Detection](./full/power-app-custom-vision/README.md)
-
-  </details>
 
   <details>
-    <summary>Advanced</summary>
+    <summary>Avanzado</summary>
 
-
-   - [Use TensorFlow and Keras for Natural Language Processing](./full/intro-nlp-tensorflow/README.md)
-   - [Deep Learning for Computer Vision](./full/deep-learning-computer-vision/README.md)
+   - [Usa TensorFlow y Keras para el procesamiento del lenguaje natural](./full/intro-nlp-tensorflow/translations/es/README.es.md)
```

Comment:
hay que revisar todos los links. Acuerdate the estos no son vinculos absolutos, es decir, `./full/` no existe en el subdirectorio `./es`. Hay que actualizar todos los vinculos pues no funcionan.

Reply:
okidoki, deja investigo eso, gracias :D